cpp/src/bitmask/legacy_bitmask.cpp

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf.h>
+#include <utilities/cudf_utils.h>
+
+gdf_size_type gdf_valid_allocation_size(gdf_size_type column_size){
+    return gdf_get_num_bytes_for_valids_allocation(column_size);
```

Comment:
```suggestion
    return gdf_get_num_bytes_for_valid_allocation(column_size);
```

Reply:
The calling function says "valid" and the callee is "valids". Be consistent.

Reply:
Come to think of it, the outer function has a much better name. Why not eliminate the inner function and move the implementation here?